packages/base-map/src/index.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useCallback, useEffect, useState } from "react";
-import { MapProvider, Map, MapRef } from "react-map-gl";
+import { MapProvider, Map, MapRef, MapProps } from "react-map-gl";
```

Comment:
Sort imports

Reply:
Sorted in 82cf6e389b83c136c2d8370515fdbcfead3294b6